x-pack/plugins/ml/public/application/components/anomalies_table/anomalies_table_columns.js

```diff
@@ -178,9 +178,20 @@ export function getColumns(
     columns.push({
       field: 'actualSort',
       'data-test-subj': 'mlAnomaliesListColumnActual',
-      name: i18n.translate('xpack.ml.anomaliesTable.actualSortColumnName', {
-        defaultMessage: 'actual',
-      }),
+      name: (
+        <EuiToolTip
+          content={i18n.translate('xpack.ml.overview.anomalyDetection.tableActualTooltip', {
+            defaultMessage: 'The actual values in the record results.',
```

Comment:
>           defaultMessage: 'The actual values in the record results.',

The [API documentation](https://www.elastic.co/guide/en/elasticsearch/reference/master/ml-get-record.html) says "The actual value for the bucket", but I found this confusing since I thought there could be multiple record results per bucket.

Reply:
You're right, there can be multiple record results for each bucket. `actual` and `typical` refer to the values seen for the record.

Reply:
Is it worth adding `anomaly` in here - `The actual values for the anomaly record results.` ?